shared/gamedata.h

```diff
@@ -1,6 +1,9 @@
 #pragma once
 #include "global_defs.h"
 #include "structs.h"
+//consider moving these to shared?
+#include "../server/db_accessor.h"
```

Comment:
we can use 2 gamedata(1 for server 1 for client) for now

Reply:
Since staticdata is used in fight in shared,
https://github.com/td1tfx/niba2.0/blob/4d427185b9e100a974774e62771a8fe2bee72f44/shared/fight.cpp#L216-L230 should we also split fight.h and fight.cpp into 2?

Reply:
we can move setup_fightable into another file(or not) and make it templated - pass in server gamedata or client gamedata